ext/civicrm_admin_ui/ang/afsearchAssignUsersToRoles.aff.html

```diff
@@ -0,0 +1,7 @@
+<div af-fieldset="">
+  <div class="af-markup">
+    <div class="help">{{:: ts(&#39;ACLs allow you to control access to CiviCRM data. An ACL consists of an Operation (e.g. \&#39;View\&#39; or \&#39;Edit\&#39;), a set of data that the operation can be performed on (e.g. a group of contacts), and a Role that has permission to do this operation.&#39;) }} <a class="crm-doc-link no-popup" href="https://docs.civicrm.org/user/en/latest/initial-set-up/permissions-and-access-control/" target="_blank">{{:: ts(&#39;Lean more ...&#39;) }}</a></div>
```

Comment:
Something strange going on with the encoding here. I'm not sure our ts scanner will handle those escaped quotes.

Reply:
Hmm - that's what 'export' gave me but I'll change them.